apps/passport-server/src/application.ts

```diff
@@ -40,10 +40,13 @@ export async function startApplication(
   logger(`[INIT] Starting application in mode ${mode}`);
 
   const dbPool = await getDB();
+  const internalPool = await getDB(5);
```

Comment:
what does the 5 do? just checking it's fine

Reply:
makes the internal pool have max 5 db connections - these are only needed intermittently